contents/zh/basics/import.md

```diff
@@ -101,6 +101,7 @@ import {
   TitleComponent,
   // 组件类型的定义后缀都为 ComponentOption
   TitleComponentOption,
+  TooltipComponent,
```

Comment:
Thanks for the catch. It seems the `TooltipOption` is also missing. Please help import it and add it `ComposeOption` as well. It would be better if you could help do the same change in the English document.

Reply:
done